blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/BlueTrendContainerImpl.java

```diff
@@ -0,0 +1,52 @@
+package io.jenkins.blueocean.service.embedded.rest;
+
+import com.google.common.base.Predicate;
+import com.google.common.collect.Iterators;
+import io.jenkins.blueocean.commons.ServiceException;
+import io.jenkins.blueocean.rest.factory.BlueTrendFactory;
+import io.jenkins.blueocean.rest.hal.Link;
+import io.jenkins.blueocean.rest.model.BluePipeline;
+import io.jenkins.blueocean.rest.model.BlueTrend;
+import io.jenkins.blueocean.rest.model.BlueTrendContainer;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.Iterator;
+
+// TODO: reenable this after refactor
+// @Restricted(NoExternalUse.class)
```

Comment:
Does leaving it cause any issue?

Reply:
Enabling this does actually cause a build issue from `enforce` since the container impl is referenced in `blueocean-pipeline-api-impl` in `BranchImpl` and `PipelineImpl`. I didn't originally add this annotation and I'm not sure if it's actually important. WDYT?